src/parser/pegen.ts

```diff
@@ -2590,6 +2590,18 @@ export function make_module(p: Parser, a: stmt[]) {
 //     }
 // }
 
+export function arguments_parsing_error(p: Parser, e: Call) {
+    let msg: string;
+    if (e.keywords.some((k) => k.arg === null)) {
+        msg = "positional argument follows keyword argument unpacking";
+    } else {
+        msg = "positional argument follows keyword argument";
+    }
+    /** @todo */
+    // @ts-ignore
+    return RAISE_SYNTAX_ERROR(msg);
```

Comment:
We'll probably have to wrap this in a `skulpt` exception when integrating in skulpt

Reply:
One thing I was thinking was we could find a place to recast the error that the parser throws. Or don't throw the error and do something similar to cpython where they have an -> error_indicator

We also need to think about the same thing with the tokeniser errors.